types/dialogflow-fulfillment/index.d.ts

```diff
@@ -0,0 +1,217 @@
+// Type definitions for dialogflow-fulfillment 0.6
+// Project: https://github.com/dialogflow/dialogflow-fulfillment-nodejs
+// Definitions by: Pascal Jordin <https://github.com/nightillusions>
+//                 SlowTick <https://github.com/slowtick>
+//                 Diego Dal Bosco <https://github.com/diegodalbosco>
+//                 Marcel <https://github.com/MTRNord>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.1
+
+import { DialogflowConversation } from 'actions-on-google';
+import { Request, Response } from 'express';
+
+export enum Platform {
```

Comment:
It looks like the underlying type (i.e. the one available at runtime) might be called "PLATFORMS": https://github.com/dialogflow/dialogflow-fulfillment-nodejs/blob/8e7b12bcea64fc3b923b1845eb4c280d610825d0/src/rich-responses/rich-response.js#L18

Reply:
(i.e. it seems like `Platform.UNSPECIFIED` won't resolve at runtime)

Reply:
I renamed `Platform` to `Platforms`

Reply:
I think it might need to be in all-caps, as in the underlying code.  You can confirm by referencing `Platforms.UNSPECIFIED` in your code and checking whether it's defined at runtime.